File: server/src/main/java/dev/slimevr/autobone/errors/AutoBoneException.java
package dev.slimevr.autobone.errors;

public class AutoBoneException extends Exception {

	public AutoBoneException() {
	}

	public AutoBoneException(String message) {
		super(message);
	}

	public AutoBoneException(Throwable cause) {
		super(cause);
	}

	public AutoBoneException(String message, Throwable cause) {
		super(message, cause);
	}

	public AutoBoneException(
		String message,
		Throwable cause,
		boolean enableSuppression,
		boolean writableStackTrace
	) {
		super(message, cause, enableSuppression, writableStackTrace);
	}
}

File: server/src/main/java/dev/slimevr/autobone/errors/AutoBoneException.kt
package dev.slimevr.autobone.errors

class AutoBoneException : Exception {
	constructor()
	constructor(message: String?) : super(message)
	constructor(cause: Throwable?) : super(cause)
	constructor(message: String?, cause: Throwable?) : super(message, cause)
	constructor(
		message: String?,
		cause: Throwable?,
		enableSuppression: Boolean,
		writableStackTrace: Boolean,
	) : super(message, cause, enableSuppression, writableStackTrace)
}
